AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -281,6 +281,7 @@ - (void)applyEnabledState:(BOOL)isEnabled {
   // Enabling.
   if (isEnabled) {
     id<MSCrashHandlerSetupDelegate> crashSetupDelegate = [MSWrapperCrashesHelper getCrashHandlerSetupDelegate];
+    MSLogDebug([MSCrashes logTag], @"Crashes setup delegate object=%@", crashSetupDelegate);
```

Comment:
"Setup crash handler with delegate"?
To highlight **the action** we're doing at the moment.

Reply:
I think @guperrot wants to see if `crashSetupDelegate` is `nil` (freed) or not at the moment. What about
```
MSLogDebug([MSCrashes logTag], @"Got crashSetupDelegate for wrapper SDK, delegate=%@", crashSetupDelegate);
```